src/awe/rmdparchive.cpp

```diff
@@ -18,41 +18,45 @@
  * along with OpenAWE. If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include <regex>
+#include <cstdint>
+#include <libintl.h>
```

Comment:
Huh... That must've been my IDE's autocompletion as I didn't put it there myself. I'll remove that and reconfigure IDE to avoid such behavior in the future.